src/assets/content/dictionary/a-b-testing.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: A/B Testing
-definition: ''
-sources: []
-perspectives: []
+definition: 'A/B testing is a UX Methodology where two variables are randomly tested on participants'
```

Comment:
Suggestion (non-blocking): I'd suggest leaving "Methodology" lowercase, as I think it's being used as a noun instead of a proper noun.